docs/changelog.rst

```diff
@@ -8,6 +8,7 @@ Changelog goes here!
 
 New features:
 
+* :doc:`/plugins/substitute`: Add the substitute plugin to solve `2786`
```

Comment:
Please provide a little description of the change introduced and then indicate `:bug:`2786` as additional information.